tensorflow/compiler/mlir/hlo/BUILD

```diff
@@ -717,6 +717,35 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
+    name = "lower_alloc_dealloc",
+    srcs = ["lib/Dialect/mhlo/transforms/lower_alloc_dealloc.cc"],
```

Comment:
Can you update the CMake file?

Reply:
mv this pattern to tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/ral_legalize_to_llvm.cc